src/pages/Detail/Detail.scss

```
@@ -0,0 +1,9 @@
+@import '../../styles/common.scss';
+
+.detailPage {
+  padding-top: 60px;
+  .detailMain {
```

```suggestion
.detailPage {
  padding-top: 60px;
  
  .detailMain {
```
새로운 셀렉터 앞에서는 한줄 띄워주세요